src/components/users/Login/Login.js

```diff
@@ -0,0 +1,155 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import Input from '../../common/Input/Input';
+import Button from '../../common/Button/Button';
+import * as actions from '../../../actions';
+
+export class Login extends Component {
+  state = {
+    loginForm: {
+      email: {
+        elementType: 'input',
+        elementConfig: {
+          type: 'email',
+          placeholder: 'Enter your email'
+        },
+        value: '',
+        labelValue: 'E-Mail Address',
+        validation: {
+          required: true,
+          isEmail: true
+        },
+        valid: false,
+        touched: false,
+      },
+      password: {
+        elementType: 'input',
+        elementConfig: {
+          type: 'password',
+          placeholder: 'Enter your password'
+        },
+        value: '',
+        labelValue: 'password',
+        validation: {
+          require: true,
+          minimum: 3
+        },
+        valid: false,
+        touched: false,
+      }
+    },
+    isFormValid: false
+  }
+
+  checkValidity = (inputValue, rules) => {
```

Comment:
Create a helpers directory ,create a validation file inside it, and put this functionality in that file so that use cane use it on other input fields in other components.